src/main/java/frc/robot/subsystems/chassis/Chassis.java

```diff
@@ -300,19 +341,18 @@ public Command createPathFollowingCommand(String path) {
     public Command createPathFollowingCommand(PathPoint... points) {
         if (points.length < 2)
             return null;
-        var trajectory = PathPlanner.generatePath(SwerveConstants.PATH_CONSTRAINTS, Arrays.asList(points));
-        return createPathFollowingCommand(trajectory, new HashMap<>(), false);
+        var trajectory = PathPlanner.generatePath(ChassisConstants.PATH_CONSTRAINTS, Arrays.asList(points));
+        return createPathFollowingCommand(trajectory, new HashMap<>(), false, true);
     }
 
     /**
      * Adds a vision input to the estimated pose of the robot
      * 
-     * @param estimatedPose     The estimated pose of the robot by vision
-     * @param timeOfMeasurement The time of the vision measurement by
-     *                          {@link Timer#getFPGATimestamp()}
+     * @param visionInput The estimated pose of the robot by vision and the time of
+     *                    the vision measurement by {@link Timer#getFPGATimestamp()}
      */
-    public void addVisionInput(Pose2d estimatedPose, double timeOfMeasurement) {
-        poseEstimator.addVisionMeasurement(estimatedPose, timeOfMeasurement);
+    public void addVisionInput(Pair<Pose2d, Double> visionInput) {
+        poseEstimator.addVisionMeasurement(visionInput.getFirst(), visionInput.getSecond());
     }
```

Comment:
should that be here?

Reply:
yes but it should be private
I notice you don't get it, the position of the robot is related to only the chassis